mw-webapp/src/component/image/Image.cy.tsx

```diff
@@ -51,4 +51,10 @@ describe("Image component", () => {
       .and("not.be.empty")
       .should("eq", LOGO_TEXT);
   });
+
+  it("should enlarge the image on click if isZoomable is true", () => {
+    cy.mount(createTestImage({src: logo}));
+    cy.get(getDataCy(IMAGE_CY)).click();
+    cy.get(`.${styles.image}`).should("be.visible"); // Verify the modal content
```

Comment:
Is it possible to check is modal opened or not by using modal's data-cy instead of style name

Reply:
Yes you can, I did that in a new commit, thanks